### PR TITLE
Reorganize and revise SES configuration chapter

### PR DESCRIPTION
**Pull request by dmpop**
bsc#1130532

### CODE REVIEW

**Comment by dmpop** [comment]
[book-installation_color_en.pdf](https://github.com/SUSE-Cloud/doc-cloud/files/4298621/book-installation_color_en.pdf)

File: xml/installation-installation-ses_integration.xml [hunk 1]
```diff
@@ -99,101 +114,32 @@
...
+      For &ses;/&ceph; deployments that have version older than 5.5, the
+      following applies. For &ceph;, it is necessary to create pools and users
+      to allow the &productname; services to use the &ses;/&ceph;
+      cluster. Pools and users must be created for &o_blockstore;,
+      &o_blockstore; backup, &o_comp; and &o_img;. Both the &o_blockstore; and
+      &o_comp; services must have the same user, as &o_blockstore; needs access
+      to create objects that &o_comp; uses.  Instructions for creating and
+      managing pools, users and keyrings is covered in the &ses; documentation
+      under <link
+      xlink:href="https://www.suse.com/documentation/suse-enterprise-storage-5/book_storage_admin/data/storage_cephx_keymgmt.html">Key
```
Comment (sknorr) [inline xml/installation-installation-ses_integration.xml]: Do note the old URL here. Please fix to: https://documentation.suse.com/en-us/ses/5.5/single-html/ses-admin/#storage-cephx-keymgmt

File: xml/installation-installation-ses_integration.xml [hunk 2]
```diff
@@ -220,51 +166,65 @@
...
+ceph.client.cinder-backup.keyring											
+ceph.client.cinder.keyring												
+ceph.client.glance.keyring					
+ses_config.yml</screen>
+
+<para>
+ Modify the <literal>glance_default_store</literal> option in <filename>~/openstack/my_cloud/definition/data/control_plane.yml</filename>:
+</para>
+
+<screen>.
```
Comment (jeremy-moffitt) [inline xml/installation-installation-ses_integration.xml]: I didnt notice it before, but this screen section looks a little weird... commenting out one line just above an identical non-commented-out duplicate of that line? the dots above the sections I assume are meant to represent a scroll of unrelated content, but I'm not sure we do that in other screen sections.

Comment (jeremy-moffitt) [reply xml/installation-installation-ses_integration.xml]: I missed the colon in the new `glance-api:` line... so maybe this is ok as-is

**Pull request merged**